modules/TerraformController.py

```diff
@@ -226,6 +230,12 @@ def dump_attack_data(self, dump_name):
                     for dump in yaml.full_load(dumps):
                         if dump['enabled']:
                             dump_out = dump['out']
+                            if last_sim:
+                                # if last_sim is set, then it overrides time in dumps.yml
+                                # and starts dumping from last simulation
+                                with open("/tmp/attack-range-%s-last-sim.tmp" % self.config['range_name'], 'r') as ls:
```

Comment:
Don't know how quoting works in Python 3, but consider doing tripple quotes (""") around "/tmp/attack-range-%s-last-sim.tmp".

This is to allow that insert %s to be anything, i.e., not be prone to quoting issues.

Reply:
This code is fine. There is no need for triple quotes as string is singled line. 

This does what you expect it to do (inserting a string at %s position)

Reply:
ok